afm/server.py

```diff
@@ -96,7 +96,7 @@ def _get_locations(self, workers):
         return locations
 
     def get_flight_info(self, context, descriptor):
-        logging.info('get_flight_info: command={}'.format(descriptor.command))
```

Comment:
I don't think you should have to send the function name.  The logger can be set to include it when it is initialized.